0000-ldap-sync-tool.md

```diff
@@ -0,0 +1,182 @@
+- Feature Name: `mgr-ldapsync`: LDAP sync tool
+- Start Date: 2019, Oct 28
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+The `mgr-ldapsync` is a tool that configures and manages users in Uyuni
```

Comment:
Can you please add a link to the current PoC code somewhere?

Reply:
Please also reference the software this RFC is intended to replace, which is:

https://github.com/uyuni-project/uyuni/blob/master/utils/sw-ldap-user-sync

Reply:
Added.